bootstrap/bootstrap-pod.yaml

```diff
@@ -17,6 +17,7 @@ spec:
       - "--enable-default-cluster-version=false"
       - "--v=4"
       - "--kubeconfig=/etc/kubernetes/kubeconfig"
+      - "--cluster-profile={{.ClusterProfile}}"
```

Comment:
We might want to add a condition like this. It will prevent to add the default value in the manifest and spread this flag too much.
``` 
{{if ne .ClusterProfile "default" }}
      - "--cluster-profile={{.ClusterProfile}}"
{{end}}
```
WDYT @wking ?

Reply:
I'm fine as this stands.  One less place to hard-code `default`